src/main/kotlin/com/example/wordle/stats/service/StatsService.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.wordle.stats.service
+
+import com.example.wordle.stats.domain.PlayerStats
+import com.example.wordle.stats.repository.StatsRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.*
+
+@Service
+class StatsService(
+    private val repo: StatsRepository
+) {
+    
+    /**
+     * 게임 결과 기록
+     */
+    @Transactional
```

Comment:
```suggestion
    @Transactional()
```
어노테이션 사용할 떄 뒤에 `()` 이 붙어야 하는 걸로 알고 있는데.. 할루시네이션일 수도 있으니 확인해주세요

Reply:
코틀린에서는 인자가 없는 어노테이션의 경우 ()을 생략하고 사용할 수 있다고 합니다.